### ISSUE TITLE
Problem with atom_indices in 1.8

### ISSUE THREAD

**Issue by CharlieLaughton** [mdtraj/mdtraj#1212]
Hi,

The following works fine with 1.7.2:
```
>>> import mdtraj as mdt
>>> u = mdt.load('2ozq.dcd', top='2ozq.pdb')
>>> sel = u.topology.selection('name CA')
>>> uCA = mdt.load('2ozq.dcd, top='2ozq.pdb', atom_indices=sel)
```

However with 1.8 I get:
```
>>> uCA = mdt.load('2ozq.dcd', top='2ozq.pdb', atom_indices=sel)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/users/charlie/.local/lib/python2.7/site-packages/mdtraj/core/trajectory.py", line 372, in load
    kwargs["top"] = _parse_topology(kwargs["top"], **topkwargs)
  File "/users/charlie/.local/lib/python2.7/site-packages/mdtraj/core/trajectory.py", line 169, in _parse_topology
    _traj = load_frame(top, 0, **kwargs)
  File "/users/charlie/.local/lib/python2.7/site-packages/mdtraj/core/trajectory.py", line 311, in load_frame
    return loader(filename, frame=index, **kwargs)
  File "/users/charlie/.local/lib/python2.7/site-packages/mdtraj/formats/pdb/pdbfile.py", line 159, in load_pdb
    assert coords.ndim == 3, 'internal shape error'
AssertionError: internal shape error
>>> 
```
Files 2ozq.dcd and 2ozq.pdb attached. Hope you can help!
[Archive.zip](https://github.com/mdtraj/mdtraj/files/591886/Archive.zip)

**Comment by mpharrigan** [comment]
Is this fixed with #1227 ?  I can reproduce with version 1.8 but it looks fixed on master

**Comment by mpharrigan** [comment]
Try this with the new release of version 1.9 and re-open if the problem persists
